app/operations/story_operations/create.rb

```diff
@@ -0,0 +1,51 @@
+module StoryOperations
+  class Create
+    include Dry::Monads[:result, :do]
+
+    def call(story:, current_user:)
```

Comment:
I think we miss this during refactoring, but why not use `story` and `current_user` as arguments for an initialize? This way you can define an `attr_reader` for both attributes and use it without having to pass them as arguments for the other methods. What do you think?

Reply:
I realized that after refactoring some operations and I agree, but let it for later. I can do that at the next PR, since it will be about abstractions to the operations.
But wont it cause any side effects to readability, or the same issues we had before the refactor?

Reply:
I would leave it to the next refactor as well. Also, before that next refactor, let's have a call to decide on some patterns, what do you think?

Reply:
Agreed